emperor/support_files/js/sceneplotview3d.js

```diff
@@ -162,13 +168,86 @@ define([
       [range.min[x], range.min[y], range.max[z]]
     ];
 
+    for (var i = 0; i < 3; i++){
```

Comment:
Not strong feelings but I think we are using both in different places. However, it seems like if we are doing something with x, y, z we are being explicit and if we are doing 1, 2, 3 we are using for loops.